### PR TITLE
[QE] fix can't catch error for "oc login"

### PR DESCRIPTION
**Pull request by lilyLuLiu**
**Fixes:** Issue #4101

### CODE REVIEW

File: test/extended/util/util.go [hunk 1]
```diff
@@ -269,7 +269,14 @@ func LoginToOcCluster(options []string) error {
 		ocLoginAsAdminCommand = ocLoginAsAdminCommand + " " + option
 	}
 
-	return ExecuteCommand(ocLoginAsAdminCommand)
+	err = ExecuteCommand(ocLoginAsAdminCommand)
```
Comment (cfergeau) [inline test/extended/util/util.go]: I think the code you add is similar to what `return ExecuteCommandSucceedsOrFails(ocLoginAsAdminCommand, "succeeds")`  does?

Comment (lilyLuLiu) [reply test/extended/util/util.go]: Thanks for pointing this out. I have re-commit the code change.

**Review by cfergeau — APPROVED** [review]

**Pull request merged**